src/home/rendering/render-github-issues.ts

```diff
@@ -17,8 +17,14 @@ export function renderGitHubIssues(tasks: TaskMaybeFull[]) {
 
   let delay = 0;
   const baseDelay = 1000 / 15; // Base delay in milliseconds
-
   for (const task of tasks) {
+    const isBot = task.preview.user?.type === "Bot"
+    const userId = task.preview.user?.id
+
+    if (isBot && userId !== 113181824 && userId !== 76412717 && userId !== 165700353 && userId !== 133917611) {
+      continue;
+    }
+
```

Comment:
Quick fix but we should remember to implement correctly asap.

Reply:
I took the 3 IDs at the end from your [comment](https://github.com/ubiquity/devpool-directory-bounties/issues/30#issuecomment-2292647684)

The first one is the one I obtained from the user endpoint for "ubiquibot[bot]" and one that is found in the `preview` objects.

This logic is saying "if it's a bot and the userId isn't one of our bots back out"

Reply:
> You must authenticate with an access token authorized to a GitHub App in order to list installations - https://docs.github.com/rest/apps/installations#list-app-installations-accessible-to-the-user-access-token

So we'd need to store our bot private key here so we can verify that way you proposed for the cron job? 

I think it's overkill for something that will be removed in no time at all once the cron is completed.

Reply:
Is this logic correct? I mean that if an issue in https://github.com/ubiquity/devpool-directory is created by a user then `isBot = false` and the `if (isBot && userId !== 113181824 && userId !== 76412717 && userId !== 165700353 && userId !== 133917611)` expression evaluates to `false` thus rendering an issue by "unauthorized" user. While the expected behavior is to skip rendering such github issues.

Overall it should be simply filtered on the backend instead in order to omit such dirty hacks.

Reply:
I've removed all issues created by `testbot6` in https://github.com/ubiquity/devpool-directory/issues so this check seems to be redundant:
```
    if (isBot && userId !== 113181824 && userId !== 76412717 && userId !== 165700353 && userId !== 133917611) {
      continue;
    }
```

Again, it's better to simply fix it on the backend https://github.com/ubiquity/devpool-directory-bounties/issues/31

Reply:
> Overall it should be simply filtered on the backend instead in order to omit such dirty hacks.

Agreed.

> Is this logic correct? 

In terms of preventing bots yeah but I hadn't considered that user's can create issues too

> Again, it's better to simply fix it on the backend https://github.com/ubiquity/devpool-directory-bounties/issues/31

So I should remove these checks here and we address the backend immediately instead